primitives/src/lib.rs

```diff
@@ -11,8 +11,8 @@ pub use self::{
     balances_map::{BalancesMap, UnifiedMap},
     big_num::BigNum,
     campaign::{Campaign, CampaignId},
-    channel::{ChannelId, ChannelSpec, SpecValidator, SpecValidators},
-    channel_v5::Channel,
+    channel::Channel,
+    channel::ChannelId,
```

Comment:
Not particularly important but this can be 
```suggestion
    channel::{Channel, ChannelId},
```

Reply:
:+1: Fixed